assets/javascripts/omise-payment-atome.js

```diff
@@ -16,25 +16,37 @@
 		$("html, body").animate({ scrollTop:0 },"slow");
 	}
 
+    function omiseHanldeError(message) {
+        $form.block({
+            message: null,
+            overlayCSS: {
+                background: '#fff url(' + wc_checkout_params.ajax_loader_url + ') no-repeat center',
+                backgroundSize: '16px 16px',
+                opacity: 0.6
+            }
+        });
+        showError(message);
+        $form.unblock();
+        return false;
+    }
+
 	function omiseFormHandler() {
         hideError();
 		if ($('#payment_method_omise_atome').is(':checked')) {
             if ($("#omise_atome_phone_default").is(":checked")) {
                 return true;
             }
+
+            const phoneNumber = $('#omise_atome_phone_number').val();
 			
-            if (!$('#omise_phone_number').val()) {
-                $form.block({
-                    message: null,
-                    overlayCSS: {
-                        background: '#fff url(' + wc_checkout_params.ajax_loader_url + ') no-repeat center',
-                        backgroundSize: '16px 16px',
-                        opacity: 0.6
-                    }
-                });
-                showError('Phone number is required in Atome');
-                $form.unblock();
-                return false;
+            if (!phoneNumber) {
+                return omiseHanldeError('Phone number is required in Atome');
+            }
+
+            const phonePattern = /(\+)?([0-9]{10,13})/;
+
+            if (!phonePattern.test(phoneNumber)) {
+                return omiseHanldeError('Phone number should be a number in Atome');
```

Comment:
Can we use `Invalid phone number` instead of `Phone number should be a number in Atome` ? Since the function did not check phone number is in Atome.

Reply:
The function retrieves the phone number from Atome and `Invalid phone number` would confuse user with billing phone number.